napari/layers/base/base.py

```diff
@@ -38,7 +37,6 @@
     get_layers,
 )
 from napari.utils.events import EmitterGroup, Event, EventedDict
-from napari.utils.events.event import WarningEmitter
```

Comment:
It is required now

Reply:
> It is required now

Readded, as well as the warning imports.